flake.nix

```diff
@@ -25,51 +25,41 @@
     flake-utils.lib.eachSystem [ "x86_64-linux" "aarch64-linux" ]
       (system:
         let
+          pkgs = import nixpkgs { inherit system; };
```

Comment:
I am finally learning about nix. Can this just be `pkgs = nixpkgs.${system}`?

Reply:
I misunderstood what this type is. The `nixpkgs` input is actually a set that represents that given `nixpkgs` module on disk (and it has `outPath` set to something like "/nix/store/...". It needs the system parameter set, and the import makes perfect sense.